Chapters/4-2-Chapter24.md

```diff
@@ -0,0 +1,211 @@
+
+"Laen! You're alive!"
+
+He hesitated, judging the voice against my motion, but then he sniffed a little and he at once he knew. He instantly let out a loud "Woop!" and rushed toward me with his big fuzzy face and his breath of stale fish. I hugged him warmly at his neck and he plopped an ear on my head while he coo'd his relief.
+
+Behind, Pomri cried out in an exhausted voice, "Laen? Is it really you?"
+
+But Pomri was not one to celebrate before it was due. She asked him, "Laen, who is that maman?", and he replied with an undulating "wOOooOOoo", a sound of sorrow, not of fear. I took a look at the pendant that was tied up in his chin fur and felt for its telltale shapes and edges. It felt familiar in my fingers, much like Kipnu's did upon my own neck.
+
+"Whoever thi is," I said, "Thi knows Melna."
+
+"Then I bet that other saffa is Boon," she said.
+
+After a pause, she called us to action again. "Come, let's get out of this cove."
+
+We followed Laen out onto beach where the light seemed brighter and bluer than before, a trick of the eyes. The smell of soot and ash hung on the air, a harbinger of the dark cloud that towered over Heiko. High above, it lofted into the coldwinds and hotwinds, spreading both toward and away from Ara in shades of gray ochre. Perhaps bits of it lifted into stellar space, reaching out for Liffa. But there we stood, at that imperceptable edge between dark and light, life and death, and we were in it.
+
+"We'll lose our bearings in this dust," said Pomri, "Let's head to the road."
+
+---
+
+Our feet took us to the traveling path and we soon reached the 1 lao'ka marking rock where we got our bearings. While Pomri and I looked for tracks, Laen had chosen a spot at the edge of the path, trying to find warmth in the thin light. He had begun to sneeze a bit and I looked back toward him, concerned.
+
+"Cover your face," she told me, holding a shawl. I wrapped the shawl over myself and waited for her to do the same with a piece of fabric ripped from her coat. I couldn't help but look at Laen again.
+
+"I don't have anything that could protect him," Pomri explained, "He'll have to stay here."
+
+"I know... I just wanted to say goodbye."
+
+"Make it quick. We still have quite a hike before we reach the pier."
+
+I took the shawl from her hands and walked over to Laen with his ears perked up in anticipation. His gaze was trained upon me, yearning for answers that I couldn't find the words to explain.
+
+From a satchel, I pulled a morsel of roasted seaweed I had brought from home and offered it to him.
+
+"You hungry?"
+
+He wasn't, but he took it gently out of respect and placed it on his left paw for later.
+
+"WoOoo," he said in a grateful tone.
+
+I found the spot on his back where he loved to be scratched and ran my fingers through his fur. His reflexes shook him with satisfaction. Then I stopped and laid my head and hands against him and he rested his head on the ground and tilted it to the side to better hear me.
```

Comment:
> His reflexes shook him with satisfaction.

No particular reason to be passive here. Make LL the subject. "He reflexively shook with satisfaction."